web/ui/dashboard/src/app/private/pages/dashboard/dashboard.component.html

```diff
@@ -146,6 +146,30 @@ <h2>Configuration</h2>
 									</option>
 								</select>
 							</div>
+							<div class="status_container">
+								<button class="filter--button" (click)="toggleShowDropdown()">
+									Status
+									<img src="/assets/img/angle-arrow-down.svg" alt="arrow down icon" />
+								</button>
+								<div class="dropdown active" *ngIf="showDropdown">
+									<div>
```

Comment:
@krisblarq Please set the elements in this `div` to stand along the same line. You can simply use:
```
display: flex;
align-item: center;
```